### ISSUE TITLE
Playlist update only introduce fields to change

### ISSUE THREAD

**Issue by AntonioMrtz** [AntonioMrtz/SpotifyElectron#246]
## Description

We want playlist update method to only need as parameters the fields that are going to change. Separate logic for adding and removing songs into two new other endpoints and methods

## Context

Frontend is required to fetch playlist data and then send a request to update playlist data, that includes both data that won't change and data that has to change

## How to do it

* Create add songs to playlist
* Create delete songs from playlist
* Update playlist rename to update_playlist_metadata. This will only require the parameters that the user want to change( Ex: no need to provide photo if you don't want to change it). The allowed fields to change with this method will be: new_name, description and photo
* Adapt frontend @AntonioMrtz 

## Testing

* Create tests for all new methods
* Adapt update playlist tests to fit update_playlist_metadata requirements

**Comment by AntonioMrtz** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Playlist update only introduce fields to change](https://trello.com/c/hELL2EPg/373-playlist-update-only-introduce-fields-to-change)

**Comment by KristianLengyel** [comment]
Hello is it possible to contribute to this issue ? I see that it is interesting topic and problem to solve.

**Comment by AntonioMrtz** [comment]
Hi @KristianLengyel , all yours. Thanks for your time, we have a [docs website](https://antoniomrtz.github.io/SpotifyElectron/) for doing the set up and other related topics. Let me know if you need anything :)

**Comment by KristianLengyel** [comment]
Just a little heads up, I'm just setting up the environment and familiarizing myself with the docs. I'll start working on the issue shortly. If I have any questions, I'll reach out.

**Comment by AntonioMrtz** [comment]
Yeah @KristianLengyel , no problem , take all the time you need, there's no need to hurry. Feel free to ask if you think I can help you with anything :)

**Comment by AntonioMrtz** [comment]
Hi @KristianLengyel are you still working on this?

**Comment by oludaara** [comment]
Can I have a go at this issue seems interesting

**Comment by AntonioMrtz** [comment]
Hi @oludaara , I will asign you :)
